src/app/units/states/portfolios/portfolios.tpl.html

```diff
@@ -43,7 +43,6 @@ <h4 class="panel-title">Mark Portfolios</h4>
               <span
                 class="grade-icon"
                 tooltip="Show all portfolios"
-                class="text-{{$index == filterOptions.selectedGrade ? 'primary' : 'muted'}}"
```

Comment:
again, why are we removing all the additional styling?

Reply:
I removed the remaining additional styling because the only component that was using it was the task-sheet-viewer component, which has now been removed as it wasn't being used. (See [line 26](https://github.com/doubtfire-lms/doubtfire-web/pull/541/commits/59d27c1e6a159fc6819ab7f7e8729e2ecefee9e6#diff-112e00f808f1a2a2115b8f236dd1d8850bf88fca11270d44a837b399afacf0e1))